modules/virt-accessing-vmi-ssh.adoc

```diff
@@ -4,51 +4,84 @@
 
 :_content-type: PROCEDURE
 [id="virt-accessing-vmi-ssh_{context}"]
-= Accessing a virtual machine instance via SSH
+= Accessing a virtual machine via SSH by using virtctl
 
-You can use SSH to access a virtual machine (VM) after you expose port 22 on it.
+You can use the `virtctl ssh` command to forward SSH traffic to a virtual machine (VM) by using your local SSH client.
 
-The `virtctl expose` command forwards a virtual machine instance (VMI) port to a node
-port and creates a service for enabled access. The following example creates
-the `fedora-vm-ssh` service that forwards traffic from a specific port of cluster nodes to port 22 of the `<fedora-vm>` virtual
-machine.
+[NOTE]
+====
+Heavy SSH traffic on the control plane can slow down the API server. If you regularly need a large number of connections, use a dedicated Kubernetes `Service` object to access the virtual machine.
+====
```

Comment:
Should we add a link/steps how to achieve this.

Reply:
If we do not already document how to create a dedicated service for a VM, that would be useful info. It is out of the scope of this task, however.